openstack/tempest/nova-tempest/values.yaml

```diff
@@ -1,5 +1,5 @@
 # You can override subchart values here, e.g. concurrency
 run_pattern: tempest.api.compute
-image_ref: cirros-vmware
-image_ref_alt: ubuntu-16.04-amd64-vmware
+image_ref: ubuntu-18.04-amd64-vmware
+image_ref_alt: ubuntu-20.04-amd64-vmware
```

Comment:
I think this is will not work for Concource CI pipeline we have to change image version on the way inside Argo manifests.